include/SFML/System/Utf.inl

```diff
@@ -38,9 +38,18 @@
 template <typename InputIt, typename OutputIt>
 OutputIt priv::copy(InputIt first, InputIt last, OutputIt d_first)
 {
+#if defined(__clang__)
+    #pragma clang diagnostic push
+    #pragma clang diagnostic ignored "-Wsign-conversion"
+#endif
```

Comment:
Is it a good idea to silence this warning for all usages of this algorithm?

I feel like it's not the algorithm that's the problem here.

Reply:
All the usages of this algorithm are limited to this file. I would "fix" the warning by saying:

```cpp
    while (first != last)
        *d_first++ = static_cast<decltype(*d_first)>(*first++);
```

but I cannot use C++11 here and I don't want to get `iterator_traits` involved.

Reply:
Oh, you can use C++11, since this is a backmerge into the master branch. You can't use it on the 2.6.x branch, but that change doesn't exist there

Reply:
Oh, I had it in my head that I was limited to C++03 for this PR. Sorry. I will try removing the pragmas.